docker/nodejs/express/app.js

```diff
@@ -23,8 +24,7 @@ app.get("/foo", function(req, res) {
 });
 
 function foo_route () {
-    var span = apm.buildSpan()
-    span.start('app.foo')
+    var span = apm.startSpan('app.foo', 'custom')
```

Comment:
`custom` type is the default, so no need to set it 😃 

```suggestion
    var span = apm.startSpan('app.foo')
```

Reply:
I was going with "let's be explicit in tests". OK to leave it in?

Reply:
Up to you 👍